src/main/java/org/Olivia/EventDispatcher.java

```diff
@@ -0,0 +1,221 @@
+package org.Olivia;
+/**
+ * Listens command from a given UiHandler
+ * parse the command and execute the corresponding function
+ * syncs with hard disk after each execution
+ * @author ZHANG TONGJUN (albertZhangTJ)
+ */
+
+import org.Olivia.IO.FileHandler;
+import org.Olivia.IO.UiHandler;
+import org.Olivia.calendar.*;
+import java.io.File;
+import java.security.InvalidParameterException;
+import java.util.List;
+
+
+public class EventDispatcher {
+
+    private Calendar table;
+    private UiHandler ui;
+    private FileHandler disk;
+
+    public EventDispatcher(Calendar table, UiHandler ui, FileHandler disk) {
+        this.table=table;
+        this.ui=ui;
+        this.disk=disk;
+    }
+
+    private int help() {
+        ui.cout(UiHandler.generateSection(UiHandler.generateHelpMsg()));
+        return 200;
+    }
+    private int list() {
+        ui.cout(UiHandler.generateSection(this.table.toString()));
+        return 200;
+    }
+
+    @Deprecated
```

Comment:
I think if you plan to use this tag, you might want to explain what function replaces this depreciated one (https://www.baeldung.com/java-deprecated)